src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntity.php

```diff
@@ -26,6 +26,7 @@ class UniqueEntity extends Constraint
     public $message = 'This value is already used.';
     public $service = 'doctrine.orm.validator.unique';
     public $em = null;
+    public $repository = null;
```

Comment:
Indeed, as I said, I was not satisfied by the name of this option. If you think `$class` is a better candidate, let's do it. 👍 

I did not use this name at first because the idea is to select and fix the repository to use for the `UniqueEntity` constraint, and I thought at the moment it could be misleading as well to have a `class` option when you already set this on an entity (`use_repository_of="Symfony\Bridge\Doctrine\Tests\Fixtures\Person"` was probably conveying this idea better, but is an hideous name 😅 ).